Makefile

```diff
@@ -92,12 +92,6 @@ lint-server:
 lint-client:
 	cd client && $(MAKE) lint
 
-# CREATING DISTRIBUTION RELEASE
-
-.PHONY: pydist-server
-pydist-server: build-server
-	cd $(BUILDDIR); python setup.py sdist -d ../dist
-	@echo "done"
```

Comment:
Pretty sure this is only for packaging the desktop (cli) version. Correct me if I'm wrong @MDunitz